internal/poolfile/config.go

```diff
@@ -188,6 +189,15 @@ func ConfigPoolFile(filepath, providerType string, conf *config.EnvConfig) (pool
 	return pool, err
 }
 
+func PrintPoolFile(pool *config.PoolFile) {
+	marshalledPool, marshalErr := yaml.Marshal(pool)
+	if marshalErr != nil {
```

Comment:
Do we really want to do this if we are adding secrets in the pool file?

Reply:
writing your first pool file is not easy. 
so this way we will want to print out the pool file, so they can copy paste it. it is only called by the setup command for now.